devops/girder/plugins/AnnotationPlugin/upenncontrast_annotation/server/helpers/connections.py

```diff
@@ -12,32 +13,40 @@ def pointToPointDistance(coord1, coord2):
     Returns:
         Number: Squared distance between point1 and point2
     """
-    return math.sqrt(
-      math.pow(coord1["x"] - coord2["x"], 2) + math.pow(coord1["y"] - coord2["y"], 2) + math.pow(coord1["z"] - coord2["z"], 2)
-    )
+    squareDist = math.pow(coord1["x"] - coord2["x"], 2) + math.pow(coord1["y"] - coord2["y"], 2)
+    if ("z" in coord1 and "z" in coord2):
+      squareDist += math.pow(coord1["z"] - coord2["z"], 2)
+    return math.sqrt(squareDist)
   
 
 def simpleCentroid(listCoordinates):
     """Compute the simple centroid of a polygon using its list of coordinates.
     The centroid corresponds to the barycenter of the polygon.
+    Compute centroid for z coordinate iff all points have z
```

Comment:
iff -> if

Reply:
"iff" means "if and only if", should I replace it with "if and only if"?

Reply:
good to know.